### PR TITLE
Feature/change password

### PR DESCRIPTION
**Pull request by sashajke**

### CODE REVIEW

File: server/src/main/java/gotcha/server/Service/Facade.java
Comment (amitmosk) [inline server/src/main/java/gotcha/server/Service/Facade.java:1]: Add TODO to line 326 : change for configuration data

File: server/src/main/java/gotcha/server/Utils/Utils.java
Comment (amitmosk) [inline server/src/main/java/gotcha/server/Utils/Utils.java:1]: minLength & maxLength should be set by configuration

**Review by amitmosk — CHANGES_REQUESTED** [review]

File: server/src/main/java/gotcha/server/Utils/Utils.java
Comment (sashajke) [reply server/src/main/java/gotcha/server/Utils/Utils.java:1]: done

File: server/src/main/java/gotcha/server/Service/Facade.java
Comment (sashajke) [reply server/src/main/java/gotcha/server/Service/Facade.java:1]: done

**Review by amitmosk — APPROVED** [review]

**Pull request merged**